common/src/main/java/org/mcaccess/minecraftaccess/features/point_of_interest/BuiltinEntityPOIGroups.java

```diff
@@ -72,6 +69,11 @@ public enum BuiltinEntityPOIGroups {
                 boolean pickupAllowedProjectile = entity instanceof AbstractArrow projectile && projectile.pickup.equals(AbstractArrow.Pickup.ALLOWED);
                 return itemOnGround || pickupAllowedProjectile;
             }
+    )),
+    Display(new POIGroup<>(
```

Comment:
Enum constants should be in SCREAMING_SNAKE_CASE.
```suggestion
    DISPLAY(new POIGroup<>(
```